CMakeLists.txt

```diff
@@ -32,59 +32,39 @@ add_definitions(
 )
 
 if(EXISTS ${PROJECT_SOURCE_DIR}/FrameworkInternals/build_config_selector.cmake)
-	  include( ${PROJECT_SOURCE_DIR}/FrameworkInternals/build_config_selector.cmake )
+    include( ${PROJECT_SOURCE_DIR}/FrameworkInternals/build_config_selector.cmake )
 else()
-     message(FATAL_ERROR "Build configuration has not been chosen yet. Please run 'quasar.py set_build_config'")
+    message(FATAL_ERROR "Build configuration has not been chosen yet. Please run 'quasar.py set_build_config'")
 endif()
 
 include( ProjectSettings.cmake )
 
+if( ( NOT DEFINED IGNORE_BOOST_SETUP ) OR ( IGNORE_BOOST_SETUP STREQUAL "OFF" ) )
```

Comment:
Please can variable `IGNORE_BOOST_SETUP` be changed to `IGNORE_DEFAULT_BOOST_SETUP` or something like that. Otherwise the variable is rather misleading, since no matter what to build a quasar server there must be _some_ boost setup. The question is rather whether the developer opts for the standard boost resolution, or uses a handmade one.

Reply:
otherwise looks good to me.

Reply:
CAEN server builds on windows too with these changes (needs one line change to build configs to instruct main CMakeLists.txt to ignore default boost setup)